roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -362,6 +374,15 @@ spec:
             items:
               - key: receptor_conf
                 path: receptor.conf
+{% if projects_persistence|bool %}
+        - name: "{{ ansible_operator_meta.name }}-projects"
+          persistentVolumeClaim:
+{% endif%}
+{% if projects_existing_claim %}
```

Comment:
`{% if projects_existing_claim %}` should be in the `{% if projects_persistence|bool %}`.
Not tested yet but in this implemetation I think the deployment with `projects_persistence: false` and `projects_existing_claim: <PVC>` gemerates broken yaml and causing error. Of course this is invalid configuration but ignoring `projects_existing_claim` if `projects_persistence: false` is safer.